Documentation.md

```diff
@@ -1,7 +1,7 @@
 # DOCUMENTATION
-Documentation is for compiling, running and debugging ringOS.
+This documentation is intended to guide you in developing, compiling and running ringOS
```

Comment:
This guide also talk about debugging, put debugging backk

Reply:
I just updated the file to include "debugging"